### PR TITLE
Refactored Card Component to functional Component 

### LINKED ISSUE #146: Refactoring Card.js from class to functional component with hooks

### PR DESCRIPTION
**Pull request by harshrai654**
Closes #146

### CODE REVIEW

**Review by jayeshchoudhary — APPROVED** [review]

File: src/features/board/components/Card.js [hunk 1]
```diff
@@ -1,74 +1,71 @@
...
+import React, { useState } from 'react';
 import { LABELS } from '../../../core/constants';
 import { GrayButton } from '../../../shared/components/Button';
 import { CardBlock, Edit, TitleInput } from '../styles';
 import { Label } from './common/Label';
 
-export class Card extends Component {
-    state = {
+export function Card(props) {
+    const [state, setState] = useState({
```
Comment (bmarvinb) [inline src/features/board/components/Card.js:9]: extract these properties from state to different variables
```javascript
const [showEditIcons, setEditIcons] = useState(false) 
etc...
```

**Comment by bmarvinb** [comment]
@harshrai654 check my comment please

File: src/features/board/components/Card.js (hunk 1 above)
Comment (harshrai654) [reply src/features/board/components/Card.js:9]: So we need each property of state object as a separate state of its own?

Comment (harshrai654) [reply src/features/board/components/Card.js]: I have made required changes

**Comment by bmarvinb** [comment]
@harshrai654 merged, thanks

**Pull request merged**